spec/components/all_components_spec.rb

```diff
@@ -23,7 +23,7 @@
 
       it "has the correct class in the ERB template",
         skip: component_name.in?(%w[step_by_step_nav_related step_by_step_nav_header step_by_step_nav previous_and_next_navigation]),
-        not_applicable: component_name.in?(%w[meta_tags machine_readable_metadata google_tag_manager_script layout_for_admin]) do
+        not_applicable: component_name.in?(%w[meta_tags machine_readable_metadata google_tag_manager_script layout_for_admin table]) do
```

Comment:
Can we still add the class? It's useful for the browser extension.

Reply:
Added the `gem-c-table` class for the extension, but it's failing because is not in the `.erb`, is in the helper